bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs2/MergePlan.scala

```diff
@@ -3,6 +3,7 @@ package edu.gemini.sp.vcs2
 import edu.gemini.pot.sp._
 import edu.gemini.pot.sp.version._
 import edu.gemini.shared.util._
+import edu.gemini.shared.util.IntegerIsIntegral._
```

Comment:
I'm not opposed to this but I could use help understanding what is going on.  With

````scala
import edu.gemini.shared.util._
````
you get `IntegerIsIntegral` which provides the `Integral` instance needed by `VersionVector`, I think.  The actual methods in `Integral` aren't directly referenced here?

What was the compile error?